Description: Use app specific tile cache directory
 Fix a problem in confined apps.
 .
Author: Christian Dywan <christian.dywan@canonical.com>
Bug-Ubuntu: https://launchpad.net/bugs/1276658
Forwarded: https://bugreports.qt-project.org/browse/QTBUG-41187
Last-Updated: 2014-09-09


--- qtlocation-opensource-src-5.5.0-alpha/src/location/maps/qgeotilecache.cpp.old	2015-03-16 16:19:00.000000000 +0000
+++ qtlocation-opensource-src-5.5.0-alpha/src/location/maps/qgeotilecache.cpp	2015-03-27 07:07:08.761722068 +0000
@@ -104,8 +104,7 @@
     // rather than in each individual plugin (the plugins can
     // of course override them)
 
-    const QString basePath = QStandardPaths::writableLocation(QStandardPaths::GenericCacheLocation)
-                 + QLatin1String("/QtLocation");
+    const QString basePath = QStandardPaths::writableLocation(QStandardPaths::CacheLocation);
 
     // delete old tiles from QtLocation 5.4 or prior
     // TODO Remove cache cleanup in Qt 6
